great_expectations/experimental/datasources/sql_datasource.py

```diff
@@ -393,7 +401,7 @@ def test_connection(self, test_assets: bool = True) -> None:
             test_assets: If assets have been passed to the SQLDatasource, whether to test them as well.
 
         Raises:
-            TestConnectionError
+            TestConnectionError: If the connection test fails.
```

Comment:
Could you add `from e` to line 413 below?

Reply:
✅